src/EnergyPlus/ChillerElectricASHRAE205.cc

```diff
@@ -151,9 +151,13 @@ void getChillerASHRAE205Input(EnergyPlusData &state)
 
         auto rep_file_name = ip->getAlphaFieldValue(fields, objectSchemaProps, "representation_file_name");
         fs::path rep_file_path = DataSystemVariables::CheckForActualFilePath(state, fs::path(rep_file_name), std::string(RoutineName));
+        if (rep_file_path.empty()) {
+            ErrorsFound = true;
+        }
         std::pair<EnergyPlusData *, std::string> callbackPair{&state,
                                                               format("{} \"{}\"", state.dataIPShortCut->cCurrentModuleObject, thisObjectName)};
         tk205::set_error_handler(tk205ErrCallback, &callbackPair);
+        Btwxt::LOG_LEVEL = static_cast<int>(Btwxt::MsgLevel::MSG_WARN);
         thisChiller.Representation =
```

Comment:
One remaining issue here, if the cbor file is not found, `rep_file_path.empty()` then this line throws an exception.
The err file looks like this:
```
   ** Severe  ** getChillerASHRAE205Input: "XYZCoolSys1-Chiller-Detailed.RS0001.a205.cbor" not found.
   **   ~~~   **   Paths searched:
   **   ~~~   **     Current Working Directory: "G:\My Drive\MJW Work-Network\Defect Test Files\9662-205Chiller\EPTEMP-00000001"
   **   ~~~   **     EnergyPlus Executable Directory: "C:\MJWGit\EnergyPlus-Review1\Build\Products\Debug"
   **   ~~~   **     "epin" Environment Variable: "G:\My Drive\MJW Work-Network\Defect Test Files\9662-205Chiller"
   **   ~~~   **     INI File Directory: "G:\My Drive\MJW Work-Network\Defect Test Files\9662-205Chiller\EPTEMP-00000001\Energy+.ini"
   **   ~~~   **     "program", "dir" from INI File: "C:\MJWGit\EnergyPlus-Review1\Build\EP-Launch-EnergyPlus-Review1-Debug"
   ** Severe  ** [json.exception.type_error.302] type must be string, but is null
```
That's enough info to fix the problem, but the exception should be avoided.
@Myoldmopar It's up to you if we want this fixed for release.